README.md

```diff
@@ -199,25 +198,27 @@ spec:
       kind: Deployment
     - group: apps
       kind: StatefulSet
-  version: "4.9.4"
-  description: "WordPress is open source software you can use to create a beautiful website, blog, or app."
   assemblyPhase: "Pending"
-  maintainers:
-    - name: Kenneth Owens
-      email: kow3ns@github.com
-  owners:
-    - "Kenneth Owens kow3ns@github.com"
-  keywords:
-    - "cms"
-    - "blog"
-    - "wordpress"
-  links:
-    - description: About
-      url: "https://wordpress.org/"
-    - description: Web Server Dashboard
-      url: "https://metrics/internal/wordpress-01/web-app"
-    - description: Mysql Dashboard
-      url: "https://metrics/internal/wordpress-01/mysql"
+  applicationMetadata:
```

Comment:
"application" in the field name seems redundant since. Would also suggest something like "descriptor" rather "metadata" to highlight the purpose of this object (descriptive information). Rename field to "descriptor"?

Reply:
+1 for "descriptor"

Reply:
Yes, initially it was 'descriptor'. I change it back to it